package.json

```diff
@@ -1,42 +1,131 @@
 {
-  "name": "root",
-  "scripts": {
-    "postinstall": "npx lerna bootstrap --hoist --ci",
-    "test": "npx lerna exec npm t",
-    "build": "npx lerna run build",
-    "docs": "npx typedoc",
-    "publish:release-candidate": "npx lerna publish --allow-branch release-candidate --dist-tag next --preid rc --no-verify-access --conventional-prerelease",
-    "publish:latest": "npx lerna publish --allow-branch main --create-release github --no-verify-access --conventional-graduate",
-    "lint": "npx lerna run lint",
-    "clean": "npm run clean:deps && npm run clean:coverage && npm run clean:build-artifacts",
-    "clean:install:build": "npm run clean && npm install && npm run build",
-    "clean:deps": "npx --yes rimraf ./packages/*/node_modules ./node_modules",
-    "clean:coverage": "npx rimraf ./packages/*/coverage",
-    "clean:build-artifacts": "npx rimraf ./packages/*/lib"
-  },
-  "devDependencies": {
-    "@babel/core": "^7.16.7",
-    "@babel/plugin-proposal-class-properties": "^7.16.7",
-    "@babel/plugin-proposal-decorators": "^7.16.7",
-    "@babel/plugin-transform-modules-commonjs": "^7.16.8",
-    "@babel/preset-env": "^7.16.8",
-    "@babel/preset-typescript": "^7.16.7",
-    "@typescript-eslint/eslint-plugin": "^5.10.0",
-    "@typescript-eslint/parser": "^5.10.0",
-    "babel-jest": "^27.4.6",
-    "eslint": "^8.7.0",
-    "eslint-config-3box": "^0.4.0",
-    "eslint-plugin-jest": "^25.7.0",
-    "jest": "^27.4.7",
-    "jest-resolver-enhanced": "^1.0.1",
-    "lerna": "^4.0.0",
-    "prettier": "^2.5.1",
-    "rimraf": "^3.0.2",
-    "typedoc": "^0.22.11",
-    "typescript": "^4.5.4"
-  },
-  "prettier": "eslint-config-3box/prettier.config",
-  "dependencies": {
-    "node-fetch": "npm:@achingbrain/node-fetch@^2.6.6"
-  }
-}
+	"name": "root",
+	"devDependencies": {
+		"@babel/core": "^7.16.7",
+		"@babel/plugin-proposal-class-properties": "^7.16.7",
+		"@babel/plugin-proposal-decorators": "^7.16.7",
+		"@babel/plugin-transform-modules-commonjs": "^7.16.8",
+		"@babel/preset-env": "^7.16.8",
+		"@babel/preset-typescript": "^7.16.7",
+		"@typescript-eslint/eslint-plugin": "^5.10.0",
+		"@typescript-eslint/parser": "^5.10.0",
+		"babel-jest": "^27.4.6",
+		"eslint": "^8.7.0",
+		"eslint-config-3box": "^0.4.0",
+		"eslint-plugin-jest": "^25.7.0",
+		"jest": "^27.4.7",
+		"jest-resolver-enhanced": "^1.0.1",
+		"lerna": "^4.0.0",
+		"prettier": "^2.5.1",
+		"rimraf": "^3.0.2",
+		"typedoc": "^0.22.11",
+		"typescript": "^4.5.4"
+	},
```

Comment:
That's not right. `package.json` should not be changed this much. You sure you are using NPM v8 + Node v16?